source/common/stats/stats_impl.cc

```diff
@@ -319,47 +320,28 @@ TagProducerImpl::addDefaultExtractors(const envoy::config::metrics::v2::StatsCon
   return names;
 }
 
-void HeapRawStatDataAllocator::free(RawStatData& data) {
+// TODO(jmarantz): move this below HeapStatDataAllocator::alloc.
```

Comment:
?  Seems like a strange TODO?

Reply:
Normally it makes sense to group alloc/free together of a single class, but these got separated in a prior PR; leaving this here now for easier diffs.  Will fix later in a follow-up.

Reply:
I see. It seemed so trivial, I was a little confused as to why you didn't just do it now. Easier review is a good reason. That makes sense.